src/odemis/odemisd/modelgen.py

```diff
@@ -898,7 +986,7 @@ def get_instantiables(self, instantiated=None):
                 comps.add(n)
 
         return comps
-            
+
```

Comment:
Same error as in https://github.com/delmic/odemis/pull/1803#discussion_r654189045.
I need to fix this, but I have already updated the PR because this seems like a minor change.